contentgrid-spring-boot-actuators/build.gradle

```diff
@@ -35,3 +35,12 @@ dependencies {
 tasks.named('test') {
     useJUnitPlatform()
 }
+publishing {
+    publications.all {
+        versionMapping {
+            allVariants {
+                fromResolutionResult()
```

Comment:
Are you sure that this is necessary?

The other projects were also published without `fromResolutionResult()`, and they worked, right?

Reply:
I got it from the annotations build.gradle, but it indeed turns out to be unnecessary for this.